components/mysql/actions/update-row/update-row.mjs

```diff
@@ -0,0 +1,91 @@
+import mysql from "../../mysql.app.mjs";
+
+export default {
+  key: "mysql-update-row",
+  name: "Update Row",
+  description: "Updates an existing row",
+  type: "action",
+  version: "0.0.1",
+  methods: {
+    async updateRow({
+      table, condition, conditionValues = [], columnsToUpdate = [], valuesToUpdate = [],
+    }) {
+      if (!conditionValues.length) {
+        throw new Error("Please provide some values to match the condition");
+      }
+
+      if (!valuesToUpdate.length) {
+        throw new Error("Please provide some values to update");
+      }
+
+      if (columnsToUpdate.length !== valuesToUpdate.length) {
+        throw new Error("The number of columns doesn't match with the number of values to update");
+      }
+
+      const updates =
+        columnsToUpdate
+          .map((column) => `SET \`${column}\` = ?`)
+          .join(", ");
+
+      const options = {
```

Comment:
I would like to suggest the same thing related to sanitizing and move the previous suggested on `delete-row.mjs` :)